feg/gateway/services/swx_proxy/cache/swx_cache.go

```diff
@@ -87,19 +85,19 @@ func NewExt(interval, ttl time.Duration) (*Impl, chan struct{}) {
 }
 
 // Get retrieves one auth vector from cache if available, adjusts cache and returns the vector, returns nil otherwise
```

Comment:
shouldn't we change the comment to `retrives neededNumber auth vectos`?

Reply:
definitely - my oversight